source/overview/index.rst

```diff
@@ -2,17 +2,17 @@
+GLOBIOM provides MESSAGE with information on land use and its implications, including the availability and cost of bioenergy, and availability and cost of emission mitigation in the AFOLU (Agriculture, Forestry and Other Land Use) sector (see :ref:`globiom`). To reduce computational costs, MESSAGE iteratively queries a GLOBIOM emulator which provides an approximation of land-use outcomes during the optimization process instead of requiring the GLOBIOM model to be rerun iteratively. Only once the iteration between MESSAGE and MACRO has converged, the resulting bioenergy demands along with corresponding carbon prices are used for a concluding analysis with the full-fledged GLOBIOM model. This ensures full consistency of the results from MESSAGE and GLOBIOM, and also allows producing a more extensive set of land-use related indicators, including spatially explicit information on land use.
 
-GLOBIOM provides MESSAGE with information on land use and its implications, including the availability and cost of bioenergy, and availability and cost of emission mitigation in the AFOLU (Agriculture, Forestry and Other Land Use) sector (see :ref:`globiom`). To reduce computational costs, MESSAGE iteratively queries a GLOBIOM emulator which provides an approximation of land-use outcomes during the optimization process instead of requiring the GLOBIOM model to be rerun iteratively. Only once the iteration between MESSAGE and MACRO has converged, the resulting bioenergy demands along with corresponding carbon prices are used for a concluding analysis with the full-fledged GLOBIOM model. This ensures full consistency of the results from MESSAGE and GLOBIOM, and also allows producing a more extensive set of land-use related indicators.
+Air pollution implications of the energy system are accounted for in MESSAGE by applying technology-specific air pollution coefficients derived from the GAINS model (see :ref:`gains`). This approach has been applied the SSP process (Rao et al., 2017 :cite:`rao_2017_SSP_airpollution`). Alternatively, GAINS can be run ex-post based on MESSAGEix-GLOBIOM scenarios to estimate air pollution emissions, concentrations and the related health impacts. This approach allows analyzing different air pollution policy packages (e.g., current legislation, maximum feasible reduction), including the estimation of costs for air pollution control measures. Examples for applying this way of linking MESSAGEix-GLOBIOM and GAINS can be found in McCollum et al. (2018 :cite:`mccollum_2018_investment`) and Grubler et al. (2018 :cite:`grubler_2018_led`).
```

Comment:
I think in the second line of this paragraph there is a typo error: 

> This approach has been applied the SSP process
should be:
> This approach has been applied **to/over** the SSP process

Probably also MESSAGEix should be written as |MESSAGEix| to be consistent with the other paragraphs.

Reply:
I've committed the changes upon these two small details. I hope it is clear.